yt_dlp/extractor/wdr.py

```diff
@@ -268,7 +268,7 @@ def _real_extract(self, url):
                     (?:
                         (["\'])(?:mediaLink|wdrrPlayerPlayBtn|videoButton)\b.*?\1[^>]+|
                         (["\'])videoLink\b.*?\2[\s]*>\n[^\n]*
-                    )data-extension=(["\'])(?P<data>(?:(?!\3).)+)\3
+                    )data-extension-ard=(["\'])(?P<data>(?:(?!\3).)+)\3
```

Comment:
no harm in also matching the old pattern, just in case, yeah?
```suggestion
                    )data-extension(?:-ard)?=(["\'])(?P<data>(?:(?!\3).)+)\3
```